src/app/core/services/data.service.ts

```diff
@@ -1,9 +1,29 @@
+ interface Dishes {
+  id: number;
+  name: string;
+  image: string;
+  description: string;
+  price: number;
+}
+
+export interface Data {
+  id: number;
+  name: string;
+  image: string;
+  dishes: Dishes[];
+}
+
```

Comment:
"Data" is not a good name for an interface. It is difficult to understand what it is.

Reply:
Done